entry.sh

```diff
@@ -8,7 +8,9 @@ rm -f /etc/avahi/services/*
 mkdir -p /var/run/dbus
 rm -f /var/run/avahi-daemon/pid
 rm -f /var/run/dbus/pid
-/etc/init.d/dbus-1 start
+if [ -z "$DEBUG" ]; then
```

Comment:
Probably still needs starting, have had conversation with Cameron and he's going to test. Only comment in PR though.